Misc/NEWS.d/next/Library/2018-03-10-20-14-36.bpo-33038.yA6CP5.rst

```diff
@@ -0,0 +1,2 @@
+Using a tempfile.SpooledTemporaryFile object with gzip.GzipFile no longer
+produces an AttributeError exception. Patch by Bo Bayles.
```

Comment:
I have noticed that your name is not in Misc/ACKS in this branch. Please add it.